astropy/modeling/functional_models.py

```diff
@@ -16,7 +16,7 @@
            'Const2D', 'Ellipse2D', 'Disk2D', 'Gaussian1D', 'Gaussian2D',
            'Linear1D', 'Lorentz1D', 'RickerWavelet1D', 'RickerWavelet2D',
            'RedshiftScaleFactor', 'Multiply', 'Planar2D', 'Scale',
-           'Sersic1D', 'Sersic2D', 'Shift', 'Sine1D', 'Trapezoid1D',
+           'Sersic1D', 'Sersic2D', 'Shift', 'Sine1D', 'Cosine1D', 'Trapezoid1D',
```

Comment:
If we are going this route, it makes me wonder if we couldn't be complete with adding Tangent1D, and the inverse functions (though on the inverse, I guess sin.inverse() may be good enough; though looking at the code, am I mistaken in seeing that there is no inverse defined? If so, that seems like an omission--and yes, the inverse is not unique, but there are standard assumptions about which value to return).

Reply:
This has been done in PR #12185.